src/Cache_Command.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * Manipulate the WP Object Cache object.
+ * Manipulates the WP Object Cache object.
```

Comment:
Could we make this "Adds, removes, fetches, and flushes the WP Object Cache object."?